src/features/MovieList/index.js

```diff
@@ -1,90 +1,49 @@
+    const movie_list = popularMovies.data;
+    const { genres } = useGenres();
+    const genre_list = genres.data;
 
     return (
-        <Wrapper>
-            <MainTitle>Popular movies</MainTitle>
-            <Content>
-                <MovieNavLink to="/movie">
-                    <MovieTile movies
-                        image={poster}
-                        title="Mulan"
-                        year="2020"
-                        genres={["Action", "Adventure", "Drama"]}
-                        star={star}
-                        rating="7,8"
-                        votes="335"
-                    />
-                </MovieNavLink>
-                <MovieTile
-                    image={poster}
-                    title="Mulan"
-                    year="2020"
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-                <MovieTile
-                    image={empty}
-                    title="Mulan"
-                    genres={["Action"]}
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-                <MovieTile
-                    image={poster}
-                    title="Mulan long title long title Mulan long title long"
-                    year="2020"
-                    genres={["Action"]}
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-                <MovieTile
-                    image={poster}
-                    title="Mulan"
-                    year="2020"
-                    genres={["Action", "Adventure", "Drama"]}
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-                <MovieTile
-                    image={poster}
-                    title="Mulan"
-                    year="2020"
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-                <MovieTile
-                    image={empty}
-                    title="Mulan"
-                    genres={["Action"]}
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-                <MovieTile
-                    image={poster}
-                    title="Mulan long title long title Mulan long title long"
-                    year="2020"
-                    genres={["Action"]}
-                    star={star}
-                    rating="7,8"
-                    votes="335"
-                />
-            </Content>
-            <Pagination />
-        </Wrapper>
+        <>
+            {popularMovies.status === "loading" ? (
+                <Loading />
+            )
+                : popularMovies.status === "error" ? (
+                    "Error while fetching data from external API"
```

Comment:
We have an Error component and it needs to be applied.

Reply:
True! Totally forgot about it